ui/app/src/components/Dialogs/DialogBody.tsx

```diff
@@ -29,7 +29,7 @@ const styles = makeStyles((theme: Theme) =>
       display: 'flex',
       flex: '1 1 auto',
       flexDirection: 'column',
-      padding: theme.spacing(2),
+      padding: `${theme.spacing(2)} !important`,
```

Comment:
This needs to change to the emotion-based API

Reply:
I will work this after merge, with another pending changes